data/remnant ships.txt

```diff
@@ -317,30 +469,101 @@ ship "Gascraft"
 		"shields" 800
 		"hull" 1500
 		"required crew" 1
-		"bunks" 1
-		"mass" 80
+		"bunks" 2
```

Comment:
Why the increase in bunk size? I believe I asked this previously, but I didn't see any mission that required this increased size.

Reply:
While there's no missions that require it right now, that will be changing in the next VS mission series.

As to why change it now instead of later; because this mission is currently a block that makes it impossible to complete these early Remnant missions if the player has any other mission active that happens to have a bunk requirement. For instance, James. (and innumerable others). Since, after all, the mission will be canceled/aborted if the player takes off in a ship that doesn't have enough bunks to hold their passengers. 

Giving it a single bunk doesn't change the nature of the ship, but does relax its unintended mission-gating effect. Especially since the Remnant are almost certainly going to be done in parallel with other campaigns, not in series.

Reply:
>As to why change it now instead of later; because this mission is currently a block that makes it impossible to complete these early Remnant missions if the player has any other mission active that happens to have a bunk requirement. For instance, James. (and innumerable others). Since, after all, the mission will be canceled/aborted if the player takes off in a ship that doesn't have enough bunks to hold their passengers.

I can take the "because I need it later" excuse, but the rest of what you're saying is pointless, as there's no limit to that line of argument. What if I want to carry Katya and Ijs around? Then every ship would need two extra bunks. What if I want Katya, Ijs, and James? That's three extra bunks. What if I want to show Rek the entire galaxy along with everyone else mentioned? Whatever the case may be, "I want to always have James with me" shouldn't be an argument for giving every ship in the game at least one bunk. And regardless, it's always possible to have a ship with the bunks you need that you just keep holding position whenever you go somewhere dangerous.

Reply:
Alright, just ignore the rest and stick with the "Because I'll need it later."